src/methods.js

```diff
@@ -128,4 +128,39 @@ export default class UtilityFunction {
         }
       }));
     }
-}
\ No newline at end of file
+
+    static addBtnEditEvent = () => {
+      let previousList = null;
+      document.querySelectorAll('.edit_btn').forEach((button) => button.addEventListener('click', (event) => {
+        event.preventDefault();
+        const inputID = 'TEXT';
+        const selectedID = event.currentTarget.id;
+        let concatID;
+
+        if (!selectedID.includes('TEXT')) {
+          concatID = inputID.concat(selectedID);
+        } else {
+          concatID = selectedID;
+        }
+
+        if (previousList !== null) {
+          previousList.getElementById(concatID).removeAttribute('readonly');
+        }
+
+        const listItem = event.target.closest('li');
+        previousList = listItem;
+        const ulItem = event.target.closest('ul');
+
+        listItem.style.background = 'rgb(230, 230, 184)';
+        ulItem.style.background = 'rgb(230, 230, 184)';
+
+        document.getElementById(concatID).removeAttribute('readonly');
+        document.getElementById(concatID).focus();
+        document.getElementById(concatID).style.background = 'rgb(230, 230, 184)';
+        listItem.querySelector('.edit_btn').style.display = 'none';
+        listItem.querySelector('.delete_btn').style.display = 'block';
+      }));
+    };
+}
+
+// console.log('testing');
```

Comment:
Hi Khusniddin  👏🏽,
Good job so far! 👍🏽
## Required Changes ♻️


Summary ♨️
No Linter error ✔️
Good and descriptive PR 💯

![ ](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

Check the comments under the review.

**Optional suggestions**
<hr/>
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Kindly, remove unnecessary commented Console.log!